e2e/specs/st_color_picker.spec.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @license
+ * Copyright 2018-2020 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/// <reference types="cypress" />
+
+describe("st.color_picker", () => {
```

Comment:
Just a reminder to capture these interaction type tests in Jest as much as possible.

Then for snapshots, you can leave `e2e/scripts/color_picker.py` the way it is and do something like..

```
  it("shows widget correctly", () => {
    cy.get(".stColorPicker").should("have.length", 4);

    cy.get(".stColorPicker").each((el, idx) => {
      return cy.wrap(el).matchImageSnapshot("colorpicker" + idx);
    });
  });
```